Functions/FirebaseFunctions/Public/FirebaseFunctions/FIRHTTPSCallable.h

```diff
@@ -24,6 +24,8 @@ NS_SWIFT_NAME(HTTPSCallableResult)
 
 - (id)init NS_UNAVAILABLE;
 
+- (instancetype)initWithData:(id)result;
```

Comment:
@ryanwilson Does this need an API review?  Is there a workaround to do in the meantime? Is this part of a larger open question about exposing public initializers for testing?

Reply:
We might be able to replace this by using the Fakes infrastructure Maksym provided in `FirebaseTestingSupport`

Reply:
Yes, this would need an API review but preferably it should use the same fakes infrastructure as `FirebaseTestingSupport` provides.

Reply:
Rolled back opening up the API, and used a fake to implement the Cloud Functions tests.